### PR TITLE
Merge `dev-fr` to `main` branch for our first release.

### PR DESCRIPTION
**Pull request by sestegra**
### Describe your changes

Thanks to all the contributors
@fydrah
@Krast76
@huats
@Horgix
@guillaumebernard84
@sestegra

### Related issue number or link (ex: `resolves #issue-number`)

N/A

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by jihoon-seo — APPROVED** [review]
Thank you for your work!
And one suggestion, I recommend to use the merge method "Squash and merge" when merging French localization PRs into `dev-fr`. Please consider. 😊

**Review by Krast76 — APPROVED** [review]

**Comment by seokho-son** [comment]
Thanks to all French localization contributors. !

Thanks to all French localization contributors!
Checklist to activate localization (for maintainers):
 - [ ] Validate all files that are not in the localization directory (content/xx/*)
 - [ ] Check i18n/xx.toml exists and it is up to date
 - [ ] Check l10n configuration in config.toml
 - [ ] Check acknowledge section for initial localization contributors
 - [ ] Check Preview works

**Comment by seokho-son** [comment]
I think we need additional PRs :slightly_smiling_face:

1. PR to update upstream dev-fr branch according to the latest upstream main branch. Please open a PR using this link https://github.com/cncf/glossary/compare/dev-fr...main (no merge conflict. so I will quickly approve once the PR opened by a contributor)

2. You can say thanks for localization contributors in the content/fr/_index.md. this also needs to be in another PR. Also, content/**en**/_index.md was recently updated. If possible please update content/fr/_index.md according to the updated En document.

**Comment by seokho-son** [comment]
> Thank you for your work! And one suggestion, I recommend to use the merge method "Squash and merge" when merging French localization PRs into `dev-fr`. Please consider. 😊

Hi @jihoon-seo Thanks for the comment. :)
Do you think this PR needs to be merged with "Squash and merge" as well?
(FYI. maintainers maintain commit history. If a PR to merge dev-xx into main has too much redundant and meaningless commits, we use "Squash and merge" method)

File: content/fr/_index.md
Comment (sestegra) [reply content/fr/_index.md:41]: Ok, we will create a dedicated for updating `_index.md`

Comment (sestegra) [reply content/fr/_index.md:57]: @seokho-son `_index.md` has been updated and merged into `dev-fr`

**Review by guillaumebernard84 — APPROVED** [review]
LGTM 🚀

**Review by huats — APPROVED** [review]
LGTM ! Thanks !

**Review by seokho-son — APPROVED** [review]
LGTM ❤️

**Comment by nate-double-u** [comment]
Congratulations everyone! 🎉

**Pull request merged**
